sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -5186,100 +5189,6 @@ SUBROUTINE CLDRAD
 
       ENDIF         ! END OF LAEROPT IF-BLOCK
 
-!! Multiply by 1.E-6 to revert these fields back
-      IF (IGET(659)>0) THEN
-         GRID1=SPVAL
-!$omp parallel do private(i,j)
-         DO J = JSTA,JEND
-            DO I = ISTA,IEND
-               IF(DUEM(I,J,1)<SPVAL) GRID1(I,J) = DUEM(I,J,1)*1.E-6
-               DO K=2,NBIN_DU
-               IF(DUEM(I,J,K)<SPVAL)&
-                GRID1(I,J) = GRID1(I,J) + DUEM(I,J,K)*1.E-6
-               END DO
-            END DO
-         END DO
-         if(grib=='grib2') then
-          cfld=cfld+1
-          fld_info(cfld)%ifld=IAVBLFLD(IGET(659))
-          datapd(1:iend-ista+1,1:jend-jsta+1,cfld)=GRID1(ista:iend,jsta:jend)
-         endif
-      ENDIF
-      
-!! Multiply by 1.E-6 to revert these fields back
-      IF (IGET(667)>0) THEN
-         GRID1=SPVAL
-!$omp parallel do private(i,j)
-         DO J = JSTA,JEND
-            DO I = ISTA,IEND
-               IF(BCEM(I,J,1)<SPVAL) GRID1(I,J) = BCEM(I,J,1)
-               DO K=2,NBIN_BC
-               IF(BCEM(I,J,K)<SPVAL)&
-                GRID1(I,J) = GRID1(I,J) + BCEM(I,J,K)
-               END DO
-            END DO
-         END DO
-         if(grib=='grib2') then
-          cfld=cfld+1
-          fld_info(cfld)%ifld=IAVBLFLD(IGET(667))
-          datapd(1:iend-ista+1,1:jend-jsta+1,cfld)=GRID1(ista:iend,jsta:jend)
-         endif
-      ENDIF
-
-      IF (IGET(660)>0) THEN
-         GRID1=SPVAL
-!$omp parallel do private(i,j)
-         DO J = JSTA,JEND
-            DO I = ISTA,IEND
-               IF(DUSD(I,J,1)<SPVAL) GRID1(I,J) = DUSD(I,J,1)*1.E-6
-               DO K=2,NBIN_DU
-               IF(DUSD(I,J,K)<SPVAL)&
-                GRID1(I,J) = GRID1(I,J)+ DUSD(I,J,K)*1.E-6
-               END DO
-            END DO
-         END DO
-         if(grib=='grib2') then
-          cfld=cfld+1
-          fld_info(cfld)%ifld=IAVBLFLD(IGET(660))
-          datapd(1:iend-ista+1,1:jend-jsta+1,cfld)=GRID1(ista:iend,jsta:jend)
-         endif
-      ENDIF
-      
-      IF (IGET(699)>0) THEN
```

Comment:
@zhanglikate Do you think variable maod can be cleaned up for the same reason above?

Reply:
@WenMeng-NOAA No. As I mentioned above, we do need to output maod using 699.